tests/tests_integration/test_api/test_hosted_extractors/test_jobs.py

```diff
@@ -0,0 +1,135 @@
+        self, cognite_client: CogniteClient, one_destination: Destination, one_event_hub_source: Source
+    ) -> None:
+        my_job = JobWrite(
+            external_id=f"myJobForTesting-{random_string(10)}",
+            destination_id=one_destination.external_id,
+            source_id=one_event_hub_source.external_id,
+            format=CogniteFormat(
+                encoding="utf16",
+            ),
+        )
+        created: Job | None = None
+        try:
+            created = cognite_client.hosted_extractors.jobs.create(my_job)
+            assert isinstance(created, Job)
+            update = JobUpdate(external_id=my_job.external_id).format.set(CogniteFormat(encoding="utf16le"))
+            updated = cognite_client.hosted_extractors.jobs.update(update)
+            format_ = updated.format
+            assert isinstance(format_, CogniteFormat)
+            assert format_.encoding == "utf16le"
+            retrieved = cognite_client.hosted_extractors.jobs.retrieve(created.external_id)
+            assert retrieved is not None
+            assert retrieved.external_id == created.external_id
+
+            cognite_client.hosted_extractors.jobs.delete(created.external_id)
+
+            with pytest.raises(CogniteAPIError):
+                cognite_client.hosted_extractors.jobs.retrieve(created.external_id)
+
+            cognite_client.hosted_extractors.jobs.retrieve(created.external_id, ignore_unknown_ids=True)
+
+        finally:
+            if created:
+                cognite_client.hosted_extractors.jobs.delete(created.external_id, ignore_unknown_ids=True)
+
+    @pytest.mark.usefixtures("one_job")
+    def test_list(self, cognite_client: CogniteClient) -> None:
+        res = cognite_client.hosted_extractors.jobs.list(limit=1)
+        assert len(res) == 1
+        assert isinstance(res, JobList)
+
+    def test_update_using_write_object(
+        self, cognite_client: CogniteClient, one_destination: Destination, one_event_hub_source: Source
+    ) -> None:
+        my_job = JobWrite(
+            external_id=f"myJobForTesting-{random_string(10)}",
+            destination_id=one_destination.external_id,
+            source_id=one_event_hub_source.external_id,
+            format=CogniteFormat(
+                encoding="utf16",
+            ),
+        )
+        created: Job | None = None
+        try:
+            created = cognite_client.hosted_extractors.jobs.create(my_job)
+
+            update = JobWrite(
+                external_id=my_job.external_id,
+                destination_id=one_destination.external_id,
+                source_id=one_event_hub_source.external_id,
+                format=CogniteFormat(
+                    encoding="utf16le",
+                ),
+            )
+
+            updated = cognite_client.hosted_extractors.jobs.update(update)
+            format_ = updated.format
+            assert isinstance(format_, CogniteFormat)
+            assert format_.encoding == "utf16le"
```

Comment:
Same suggestion here
```suggestion
            updated = cognite_client.hosted_extractors.jobs.update(update)
            assert isinstance(updated.format, CogniteFormat)
            assert updated.format.encoding == "utf16le"
```